src/tools/factory-prov-tool/exp32-restart/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+#
+#   Copyright (c) 2019 Google LLC.
+#   All rights reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+#
+#   @file
+#         Makefile for esp32-restart program.
```

Comment:
Typo in the directory name (i.e., "exp32-restart" vs. "esp32-restart")?

Reply:
Will fix.